x-pack/plugins/enterprise_search/public/applications/shared/schema/errors_accordion/schema_errors_accordion.scss

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-.schemaFieldError {
+ .schemaFieldError {
```

Comment:
Was this intentional? Seems to be a 3-space indentation now

Reply:
Woops - awesome catch/eagle eyes Scotty! Definitely not intentional, but looks like I managed to not screw it up when I changed the CSS class name to [.schemaErrorsAccordion](https://github.com/elastic/kibana/pull/99069/files#diff-98d31f4b8db472bb2d9ed22334746cc41003159646ac4ea0aea304352da30cd3R8) to match the component